.env.sample

```diff
@@ -17,9 +17,5 @@ SIMPLEI_ENV_NAME=Local
 CONFIG_SYNC_DIRECTORY=/app/project/config
 FILE_PRIVATE_PATH=/app/.lando/private
 HASH_SALT=
-NW_TEST_USER_PREFIX=
```

Comment:
Not sure if these are used anymore, but we can add them if the QA team do make use of them